src/styles/card/image.scss

```diff
@@ -0,0 +1,9 @@
+.images {
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 160px;
+  height: 134px;
```

Comment:
It's better to avoid setting fixed width and height to the images. It can lead to the distortion of the image if its aspect ratio doesn't match the one you specified. It's better to set only width or height and let the browser calculate the other dimension to maintain the aspect ratio. You can set max-width or max-height to limit the size of the image.